### ISSUE TITLE
Import GC JSON file failed

### ISSUE THREAD

**Issue by eugenpp** [GoldenCheetah/GoldenCheetah#1370]
Windows 8.1 x64
Some specific dates failed to parse if operating system Locale isn't English. My Windows has locale set to Slovak and import failed. After switching to English US, file has been imported correctly.
```
  "RIDE": {
    "STARTTIME": "2014/10/16 15:39:10 UTC",
```

**Comment by Joern-R** [comment]
@eugenpp - since I am running/build GC on a German locale where date/time formats also deviate from US and on Windows (7  SP1) without a problem I had a look into the code. Actually formatting of date/time is set "hard" to the format above - both for input and output. Can you please send example .JSON files were created by GC - and do not parse correctly - for me to check  joern-rm  at  gmail.com

**Comment by eugenpp** [comment]
Summary:
Windows 8.1 x64, system locale Slovak (encoding CP1250), regional settings Slovak, GC 3.1.0 buildid=3100

1) I imported slf file from test (test/rides/Log_100502_0951.slf).
2) I changed ride date and time to 20141016T15:39:10
3) Then exported ride as GC JSON. Note that exported file is CP1250 encoded without BOM (IMO should be UTF8).

4) I deleted ride from GC.
5) Import json file....
Importing file failed. Import is partially recognized because next import of the same file failed with "Error - file exists" but is not visible.

**Comment by eugenpp** [comment]
@Joern-R 
Delivery to the following recipient failed permanently:

     joern-rm at gmail.com

550-5.1.1 The email account that you tried to reach does not exist. Please try

**Comment by eugenpp** [comment]
Month and weekday tags contains special characters, cp1250 encoded. Are they handled correctly?
```
		"TAGS":{
			...
			"Device":"Sigma ROX Log",
			"File Format":"",
			"Filename":"2014_10_16_15_39_10.json",
			"Month":"október",
			"Weekday":"št",
			"Year":"2014"
		},
```

**Comment by Joern-R** [comment]
@eugenpp Upps. sorry - email is joern.rm @ gmail. But your latest example helped me to remember the problem.

It relates to 3.10 and earlier - just checked the commits - we have fixed that problem after 3.10 with Commit https://github.com/GoldenCheetah/GoldenCheetah/commit/f3a7af9890fc5381b2557ca6c4ae78ec5c74e51a

You can test with a current DEV build - or wait for the next one coming out soon. When going for a DEV build you either need to do this on a separate installation - or stick with it until the final 3.2 is available, since we have changed the file structure significantly and there is no way back to 3.1 (besides are data recovery) - including setting back registry on Windows.
